app.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories.controller");
+
+const app = express();
+app.use(express.json());
+// console.log("Hi, I'm in app");
```

Comment:
Please remove your app.use(express.json()) at this point we you are only making a GET request and so this line is unnecessary 👀